src/python/pants/backend/python/targets/python_binary.py

```diff
@@ -43,6 +43,7 @@ def __init__(self,
                indices=None,              # pex option
                ignore_errors=False,       # pex option
                shebang=None,              # pex option
+               build=True,                # pex option
```

Comment:
Probably what you want to do for this option is have it default to "None"/unset, and then if it is not set on a target, use the global default (ftr: `PrimitiveField(None)` should work fine).